template/partials/hardware/portStatus.tmpl.partial

```diff
@@ -1,13 +1,13 @@
 <p>
-  The following excerpt from the {{{hardware}}} <a class="xref" href="~/articles/hardware/{{{oe.hardwareDirectory}}}/overview.html">example workflow</a> demonstrates port status functionality and saves timestamped port status data.
+  The {{{hardware}}} reports when a port connection enters or leaves an aberrant state. This is helpful for diagnostic purposes - knowing the time and and type of a communication failure can facilitate tracking down its cause. The following excerpt from the {{{hardware}}} <a class="xref" href="~/articles/hardware/{{{oe.hardwareDirectory}}}/overview.html">example workflow</a> demonstrates port status functionality and saves timestamped port status data.
```

Comment:
Altered text:

The {{{hardware}}} reports when a headstage port connection enters or leaves an aberrant state. This is helpful for diagnostic purposes.  Knowing the time and and type of a headstage communication failure is a good first step to facilitate tracking down its cause.